plugins/CoreHome/vue/src/ReportExport/ReportExportPopover.vue

```diff
@@ -0,0 +1,425 @@
+
+      let { date, period } = dataTable.param;
+
+      if (reportFormat === 'RSS') {
+        date = 'last10';
+      }
+      if (typeof dataTable.param.dateUsedInGraph !== 'undefined') {
+        date = dataTable.param.dateUsedInGraph;
+      }
+
+      const formatsUseDayNotRange = Matomo.config.datatable_export_range_as_day.toLowerCase();
+
+      if (formatsUseDayNotRange.indexOf(reportFormat.toLowerCase()) !== -1
+        && dataTable.param.period === 'range'
+      ) {
+        period = 'day';
+      }
+
+      // Below evolution graph, show daily exports
+      if (dataTable.param.period === 'range'
+        && dataTable.param.viewDataTable === 'graphEvolution'
+      ) {
+        period = 'day';
+      }
+
+      const exportUrlParams: Record<string, unknown> = {
+        module: 'API',
+        format: reportFormat,
+        idSite,
+        period,
+        date,
+      };
+
+      if (reportType === 'processed') {
+        exportUrlParams.method = 'API.getProcessedReport';
+        [exportUrlParams.apiModule, exportUrlParams.apiAction] = apiMethod.split('.');
+      } else {
+        exportUrlParams.method = apiMethod;
+      }
+
+      if (dataTable.param.compareDates
+        && dataTable.param.compareDates.length
+      ) {
+        exportUrlParams.compareDates = dataTable.param.compareDates;
+        exportUrlParams.compare = '1';
+      }
+
+      if (dataTable.param.comparePeriods
+        && dataTable.param.comparePeriods.length
+      ) {
+        exportUrlParams.comparePeriods = dataTable.param.comparePeriods;
+        exportUrlParams.compare = '1';
+      }
+
+      if (dataTable.param.compareSegments
+        && dataTable.param.compareSegments.length
+      ) {
+        exportUrlParams.compareSegments = dataTable.param.compareSegments;
+        exportUrlParams.compare = '1';
+      }
+
+      if (typeof dataTable.param.filter_pattern !== 'undefined') {
+        exportUrlParams.filter_pattern = dataTable.param.filter_pattern;
+      }
+
+      if (typeof dataTable.param.filter_pattern_recursive !== 'undefined') {
+        exportUrlParams.filter_pattern_recursive = dataTable.param.filter_pattern_recursive;
+      }
+
+      if (window.$.isPlainObject(requestParams)) {
+        Object.entries(requestParams).forEach(([index, param]) => {
+          let value = param;
+          if (value === true) {
+            value = 1;
+          } else if (value === false) {
+            value = 0;
+          }
+          exportUrlParams[index] = value;
+        });
+      }
+
+      if (this.optionFlat) {
+        exportUrlParams.flat = 1;
+        if (typeof dataTable.param.include_aggregate_rows !== 'undefined'
+          && dataTable.param.include_aggregate_rows === '1'
+        ) {
+          exportUrlParams.include_aggregate_rows = 1;
+        }
+      }
+
+      if (!this.optionFlat && this.optionExpanded) {
+        exportUrlParams.expanded = 1;
+      }
+
+      exportUrlParams.format_metrics = this.optionFormatMetrics ? 1 : 0;
```

Comment:
before the parameter was only set if the checkbox was checked. Guess that makes a difference, as when the value is not given it uses internally `bc` as default, which will at least format percent metrics.

Reply:
Even if it's set to `0`? I changed this since I noticed the checkbox had no effect for evolution graphs where format_metrics was set in the datatable parameters. I tried testing just now and I cant get the report export to use bc if format_metrics is set to 0, can you confirm?

Reply:
Not sure if I understood that right. But I see what you mean regarding the evolution charts. Maybe in that case we should manipulate the datatable parameters and remove those that can be set in the export popover and maybe use the value from datatable parameters as default?

Reply:
I think we can just set it based on whether the checkbox is checked, eg, checked: `format_metrics=1`, unchecked `format_metrics=0`, then it's always supplied and never defaults to `format_metrics=bc`. Would that work or am I missing something?

Reply:
But `bc` is kind of the default if no value is given. So always providing `0` will change the result of the exports (compared to the behavior before). See https://github.com/matomo-org/matomo/blob/3b00013819863472afe30cc04de49c4365c22d21/core/API/Request.php#L108-L110

Reply:
It doesn't change the result of the API just the URL in the export popover, which I would say is more useful to the user than having half metrics formatted, and the other half not. But this isn't relevant to vue so I'll just change it back.